CHANGELOG.md

```diff
@@ -4,6 +4,11 @@ Please note that compatibility for 0.x releases (software or repositories) isn't
 
 _When adding new entries to the changelog, please include issue/PR numbers wherever possible._
 
+
+## Unreleased
```

Comment:
Not for any reason, but our convention so far has been this:

## 0.15.4 (UNRELEASED)

Reply:
can change it. is the new convention just as good? Doesn't commit us to necessarily doing another point release; we could end up rolling the change into 0.16.0